src/commands/utils/typeCommand.ts

```diff
@@ -0,0 +1,122 @@
+
+export const typeCommand: CommandDefinition = {
+    name: supportedAircraftTypes,
+    description: 'Shows the command details for the specified supported aircraft type',
+    category: CommandCategory.UTILS,
+    executor: async (msg) => {
+        const { author } = msg;
+        const [dotEvokedCommand] = msg.content.trim().split(/\s+/);
+        const evokedCommand = dotEvokedCommand.substring(1);
+        let subCommand;
+        if (!supportedAircraftTypes.includes(evokedCommand)) {
+            // Executor evoked for generic Message Command, not specific type
+            subCommand = evokedCommand;
+        } else {
+            const args = msg.content.replace(dotEvokedCommand, '').trim();
+            if (!args) {
+                Logger.debug(`Type Command - evoked: ${evokedCommand} - Error: No sub command`);
+                // Evoked without an actual command
+                return;
+            }
+            [subCommand] = args.split(/\s+/);
+        }
+        const command = commands[subCommand];
+        if (!command) {
+            Logger.debug(`Type Command - evoked: ${evokedCommand} - sub: ${subCommand} - Sub command doesn't exist`);
+            // Evoked with a non-existing command
+            return;
+        }
+        const { name } = command;
+        const commandsArray = Array.isArray(name) ? name : [name];
+        if (!commandsArray.includes(subCommand)) {
+            Logger.debug(`Type Command - evoked: ${evokedCommand} - sub: ${subCommand} - Mismatch between sub command and command`);
+            // Somehow it found a command that doesn't actually support this command string?
+            return;
+        }
+        if (!isMessageCommand(command)) {
+            Logger.debug(`Type Command - evoked: ${evokedCommand} - sub: ${subCommand} - Sub Command is not a Message Command`);
+            // This isn't a simple MessageCommand
+            return;
+        }
+        const { genericEmbed, typeEmbeds } = (command as MessageCommandDefinition);
+        if (!enableMultipleAircraftTypes || !typeEmbeds || Object.keys(typeEmbeds).length === 0) {
+            if (subCommand === evokedCommand) {
+                const postEmbed = typeEmbeds && Object.keys(typeEmbeds).includes(defaultDisabledAircraftType) ? typeEmbeds[defaultDisabledAircraftType] : genericEmbed;
+                await msg.channel.send({ embeds: [postEmbed] });
+            }
+            return;
+        }
+        const commandSupportedAircraftTypes = typeEmbeds ? Object.keys(typeEmbeds) : [];
+        if (subCommand === evokedCommand) {
+            const commandSupportedAircraftTypeEmojies = [];
+            const { data: genericEmbedData } = genericEmbed;
+            const postGenericEmbed = makeEmbed((genericEmbedData as EmbedData));
+            const choiceEmbedFieldLines = [];
+            choiceEmbedFieldLines.push('Please select the appropriate reaction for the Aircraft for which you would like more information:');
```

Comment:
```suggestion
            choiceEmbedFieldLines.push('Please select the appropriate reaction for the Aircraft you would like more information about:');
```

Removes the double 'for'. Thoughts?